components/Login.tsx

```diff
@@ -12,7 +12,7 @@ interface FormValues {
   password: string;
 }
 
-const REDIRECT_URL = `${HOSTNAME}/account`;
+const REDIRECT_URL = `${HOSTNAME}/profile`;
```

Comment:
Wanted to ask you earlier: what are your thoughts re: Formik? Seems we're deep into using it, which is fine, but will it slow us down / reduce customizability in long-term?

Reply:
I think Formik's very flexible and we shouldn't be blocked by using it. Really abstracts away a lot of form logic for us and that't about it. Would love to know if @elliotfklein or @shamdasani have some specific concerns!

Reply:
Yep, I was very happy with how easy it was to make custom components that work with Formik (e.g. `EditAvatar` and `Multiselect`).

Only hitch so far has been trying to validate certain fields at different times (e.g. only do `username` `onBlur`, instead of `onChange`) -- not critical, but doesn't appear to be supported without some significant workarounds: https://github.com/jaredpalmer/formik/issues/3445 https://github.com/jaredpalmer/formik/issues/2372 https://github.com/jaredpalmer/formik/issues/512 

Current solution is to just have the `Step1` form (where we collect username) validate all fields `onBlur`, which is fine I think